docs/application/native/guides/internationalization/i18n.md

```diff
@@ -71,8 +71,12 @@ The main features of the i18n API include:
 
   You can [iterate through strings](#uchar_iter), in a safe way, with UCharIter.
 
+- Formating plural messages
+
+  You can [format plural messages](#plural_format), with PluralFormat.
+
 > **Note**  
-> The Alphabetic Index, FieldPosition, Format, Formattable, Measure, MeasureFormat, MeasureUnit, ParsePosition, Ubidi, Ushape, and Utmscale APIs are supported since Tizen 3.0. The PluralRules, Uversion, and UCharIter APIs are supported since Tizen 4.0. The Immutable Index is supported from Tizen 5.0.
+> The Alphabetic Index, FieldPosition, Format, Formattable, Measure, MeasureFormat, MeasureUnit, ParsePosition, Ubidi, Ushape, and Utmscale APIs are supported since Tizen 3.0. The PluralRules, Uversion, and UCharIter APIs are supported since Tizen 4.0. The Immutable Index and the PluralFormat is supported from Tizen 5.0.
```

Comment:
@mijin85cho, Yes it is okay to use.
The Immutable Index and the PluralFormat is supported from Tizen 5.0.
When we read the above sentence there are two formats that are supported in version 5.0.
Hence, I have used "are".
Note:  If you use "or" in a sentence we use "is".
If you use "and" in the sentence we use "are"

The reason for the usage of "from" in this sentence is that in technical document we generally avoid maximum usage of "since" in the sentence.

In this sentence 5.0 is the latest version and we will have version after that in future, so it is better use "from" in this case

I hope the question is answered?

Reply:
@sabeena21, thanks for your answer, but my point is : 

1. Your review comment is not applied. 
   This sentence is still using "is" while you're guiding to use "are".
    But you approved this PR. That's why I'm asking.

2. In this paragraph, we had "since Tizen 3.0", "since Tizen 4.0", "since Tizen 5.0".

   As you said you are avoiding "since", "since Tizen 5.0" is changed to "from Tizen 5.0".
   So, in case of "since Tizen 3.0" and "since Tizen 4.0", are they left with intend, as they are not latest version?

   `Alphabetic Index, FieldPosition, Format, Formattable, Measure, MeasureFormat, MeasureUnit, ParsePosition, Ubidi, Ushape, and Utmscale APIs` are supported from Tizen 3.0, to current version, and will to futural version.
    So I thought it will use same rule, using "from". Same issue on "since Tizen 4.0".

Reply:
@mijin85cho , 
1st : Sorry i missed out looking out on that.
2nd : It is okay to keep the sentence as it because 4.0, 3.0 are older version so it okay to retain since for this sentence.
Only for newer version we can change to from.

Reply:
2nd : I see, so in this paragraph we are going to use "since Tizen 3.0", "since Tizen 4.0", and "from Tizen 5.0". And I right?

Reply:
@mijin85cho , Yes it it is correct to use "since Tizen 3.0", "since Tizen 4.0", and "from Tizen 5.0"